qcodes/instrument_drivers/AlazarTech/ATS9360.py

```diff
@@ -77,55 +72,48 @@ def __init__(self, name, **kwargs):
                              'EXTERNAL_CLOCK': 64,
                                   'UNDEFINED': 'UNDEFINED'})
         self.add_parameter(name='clock_edge',
-                           get_cmd=None,
                            parameter_class=TraceParameter,
                            label='Clock Edge',
                            unit=None,
                            initial_value='CLOCK_EDGE_RISING',
                            val_mapping={'CLOCK_EDGE_RISING': 0,
                                         'CLOCK_EDGE_FALLING': 1})
         self.add_parameter(name='decimation',
-                           get_cmd=None,
                            parameter_class=TraceParameter,
                            label='Decimation',
                            unit=None,
                            initial_value=1,
                            vals=validators.Ints(0, 100000))
 
-        for i in ['1', '2']:
-            self.add_parameter(name='coupling' + i,
-                               get_cmd=None,
+        for i in range(1, self.channels+1):
```

Comment:
I think channels is missing to be defined.

Reply:
No the default value of 2 is set in the super class `AlazarTech_ATS`